frontend/public/components/replicaset.jsx

```diff
@@ -120,6 +128,7 @@ const Details = ({ obj: replicaSet }) => {
             <dl className="co-m-pane__details">
               <ResourcePodCount resource={replicaSet} />
               <RuntimeClass obj={replicaSet} />
+              <PodDisruptionBudgetField obj={replicaSet} />
```

Comment:
Doesn't `PodDisruptionBudgetField` add its own `dl`? We should avoid nesting `dl` elements.